modules/javafx.graphics/src/main/java/com/sun/javafx/tk/quantum/QuantumRenderer.java

```diff
@@ -54,7 +54,7 @@
 /*
  * Quantum Renderer
  */
-final class QuantumRenderer extends ThreadPoolExecutor  {
+final public class QuantumRenderer extends ThreadPoolExecutor  {
```

Comment:
I am not in favor of making this class public. I would rather add a method to Toolkit, and have the overridden method in QuantumToolkit call the Quantum renderer as needed.

Reply:
I've just updated the PR and moved it to the QuantumToolkit!